website/docs/r/aviatrix_datadog_agent.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Please document all the attributes of the resource, including `Computed` values. (`status` is missing)